src/main/java/decide/DecideHelpFunctions.java

```diff
@@ -1,9 +1,47 @@
 package decide;
 
 public class DecideHelpFunctions {
-
-    public boolean conditionFunctionZero(double length, double[][] coordinates) {
-        return true;
+	
+	// Equal, Lesser, Greater
+	// Return 1 if a > b, 0 if a = b, -1 if a < b
```

Comment:
Use proper JavaDoc syntax when commenting functions. See `conditionFunctionSeven` for example.